people/timur.json

```diff
@@ -3,9 +3,9 @@
     "name": "Timur Abragimovich",
     "nick": "Timur",
     "email": "timur.abragimovich@mild.blue",
-    "title_cs": "JUNIOR DEVELOPER",
-    "title_en": "JUNIOR DEVELOPER",
-    "title_de": "JUNIOR DEVELOPER",
+    "title_cs": "Junior vývojář",
+    "title_en": "Junior developer",
+    "title_de": "Junior Developer",
```

Comment:
Možná je to mimo scope toho pull request, ale možná by už bylo dobré i změnit tu německou verzi na Junior-Entwickler?

Reply:
Diky z apodnet, zmenil som na Entwickler (bez pomlčky, rovnako ako u ostatných)